Data-Structures/src/main/java/Tree/BinaryTree.java

```diff
@@ -53,6 +53,23 @@ public void createTree()
 
     }
 
+  //find_maximum_binary_tree
+
+    public int findmaximumvalue(Node node){
+          int cur = -9999;
```

Comment:
Integer.MIN_VALUE will assign a value the lowest int (-2147483648)
`int cur = Integer.MIN_VALUE;`

Reply:
Thank you!